tests/xlfunctions/test_logical.py

```diff
@@ -54,8 +54,11 @@ def test_TRUE(self):
         self.assertTrue(logical.TRUE())
 
     def test_IFERROR(self):
-        self.assertEqual(logical.IFERROR(10/2,0),5) # Check if value is OK
-        self.assertEqual(logical.IFERROR(10/0,0),0) # Check if value cause error
-        self.assertEqual(logical.IFERROR(10/0,"ERROR"),"ERROR") # Check returning string
-        self.assertEqual(logical.IFERROR(logical.IFERROR(10/0,0) + 1,0),1) # Check nested IFERROR
-
+        # Check if value is OK
+        self.assertEqual(logical.IFERROR(10 / 2, 0), 5)
+        # Check if value cause error
+        self.assertEqual(logical.IFERROR(10 / 0, 0), 0)
```

Comment:
This cannot possibly work. `10/0` immediately throws a `ZeroDivisionError`, because execution is not delayed.

Reply:
Thank you for your quick respond!
If I understand correctly, I should manually throw an exception in this case to test this function?